### PR TITLE
Removes all uses of a certain word in a certain sense

### PR DESCRIPTION
**Pull request by Crazylemon64**
Might as well be sure we're not running foul of any wide-net policies
Expansion of #13275

### CODE REVIEW

File: code/game/turfs/simulated/walls_misc.dm [hunk 1]
```diff
@@ -72,10 +72,9 @@
 	realappearance = new /obj/effect/clockwork/overlay/wall(src)
 	realappearance.linked = src
 
-/turf/simulated/wall/clockwork/Destroy()
+/turf/simulated/wall/clockwork/BeforeChange()
```
Comment (Crazylemon64) [inline code/game/turfs/simulated/walls_misc.dm:75]: whoops, this change is a stowaway

this makes the clockwork wall removal not leave the overlay behind

**Comment by Ty-Omaha** [comment]
"Retarded" is allowed as a synonym of idiot fwiw

**Comment by AffectedArc07** [comment]
@Ty-Omaha with what we have seen today, its better to not take chances. I would like to remind you goofball got banned from github entirely for this word.

**Comment by Ty-Omaha** [comment]
>@Ty-Omaha with what we have seen today, its better to not take chances. 

This has been ruled on by heads on multiple occassions and it is allowed.

**Comment by AffectedArc07** [comment]
@Ty-Omaha The rules on github are not the same on what the heads rule acceptable ingame. Just because you can say something ingame, doesnt mean a multimillion dollar company is going to accept it.

**Comment by Ty-Omaha** [comment]
>@Ty-Omaha The rules on github are not the same on what the heads rule acceptable ingame. Just because you can say something ingame, doesnt mean a multimillion dollar company is going to accept it.

The comment was purely in case someone used this PR as a supporting point for the argument that it is not allowed in game.

**Comment by KasparoVy** [comment]
Desperate times call for desperate PRs. Damn, brutal

**Comment by FreeStylaLT** [comment]
we word police now

**Pull request merged**